roofit/roofitcore/src/RooHistPdf.cxx

```diff
@@ -379,6 +435,18 @@ double RooHistPdf::analyticalIntegral(Int_t code,
   return ret ;
 }
 
+std::string RooHistPdf::buildCallToAnalyticIntegral(Int_t code, const char * /*rangeName */,
+                                                    RooFit::Detail::CodeSquashContext & /* ctx */) const
+{
+   if (((2 << _histObsList.size()) - 1) != code) {
+      coutE(InputArguments) << "RooHistPdf::integral(" << GetName()
+                            << ") ERROR: AD currently only supports integrating over all histogram observables."
+                            << std::endl;
+      return "";
+   }
+   return RooNumber::toString(_dataHist->sum(false));
```

Comment:
Why do we need `toString()` here? I thought we need it only for values that can be infinity, but this is not the case for these template histograms